src/main/java/com/android/volley/RequestQueue.java

```diff
@@ -160,9 +160,9 @@ public void stop() {
         if (mCacheDispatcher != null) {
             mCacheDispatcher.quit();
         }
-        for (int i = 0; i < mDispatchers.length; i++) {
-            if (mDispatchers[i] != null) {
-                mDispatchers[i].quit();
+        for (final NetworkDispatcher mDispatcher : mDispatchers) {
```

Comment:
The linter is being a bit overzealous here; this is actually a violation of the "Avoid iterators" advice in the [Developing for Android](https://medium.com/google-developers/developing-for-android-ii-bb9a51f8c8b9) doc.

That being said, given that the doc acknowledges that this isn't a big deal in practice as long as it's a single allocation (that is, not being done in an inner loop), so I'm okay with it - stop() shouldn't ever be called in a loop.